optimade.md

```diff
@@ -317,17 +321,43 @@ An example of a full response:
 | 418  | Asked for a non-existent keyword                                      |
 | 422  | Database returned (200) but the translator failed                     |
 
-## <a name="h.3.4">3.4. Index database</a>
-
-The provider MUST supply one OPTiMaDe API base URL for an "index" database. 
-This database MUST be empty.
-
-It MUST provide at least the `info` and `databases` endpoints. These do not
-need to be querable, i.e., they MAY be provided as static JSON files. However,
-they MUST return the correct and updated information on all sibling databases
-for the current provided, as the main purpose of the "index" database API endpoint
-it to allow for automatic discoverability of all sibling databases of a given 
-provider, see [appendix 1](#h.app1).
+Notes:
+* if a client receives an unexpected 404 error when making a query to a base URL, 
+  and is aware of the index meta-database that belongs to the database provider 
+  (as described in [3.4. Index meta-database](#h.3.4)),
+  the next course of action SHOULD be to fetch the index entry from the meta-database
+  corresponding to the same database id as was queried, and if this has changed, 
+  redirect its query to the new Base URL.

```

Comment:
Yes, I want to take the opportunity to add in the specification that clients are expected to not just throw "Database suddenly disappeared!"-type errors when someone moves a database. If implemented this way, I anticipate this will save us all some frustration down the line... Feel free to reformulate in a better way.